libmogwai-schedule/scheduler.c

```diff
@@ -583,6 +583,8 @@ mws_scheduler_update_entries (MwsScheduler  *self,
   actually_added = g_ptr_array_new_with_free_func (g_object_unref);
   g_autoptr(GPtrArray) actually_removed = NULL;  /* (element-type MwsScheduleEntry) */
   actually_removed = g_ptr_array_new_with_free_func (g_object_unref);
+  g_autoptr(GPtrArray) actually_active_removed = NULL;  /* (element-type MwsScheduleEntry) */
```

Comment:
Nitpick: `active_actually_removed` ? :slightly_smiling_face:

Reply:
I’ve changed it to `actually_removed_active` and will push a fixup shortly.

Reply:
:trollface: